core/src/Dscp/Core/Validation/Educator.hs

```diff
@@ -144,21 +176,29 @@ verifyStudentSubmission author ss = do
 --    for given submission
 validatePrivateBlk :: PrivateBlock -> Either [BlockValidationFailure] ()
 validatePrivateBlk pb =
-    let subs =_pbbTxs (_pbBody pb)
-        merkleRoot = getMerkleRoot (fromFoldable subs)
-        headerVer = validateHeader (_pbHeader pb) merkleRoot
+    let subs        = _pbbTxs (_pbBody pb)
+        merkleRoot  = getMerkleRoot (fromFoldable subs)
+        headerVer   = validateHeader (_pbHeader pb) merkleRoot
         headerValid = verifyGeneric headerVer
-        subValid = foldl' (\acc sub -> validateSub sub `mappendLefts` acc)
-                   pass subs
-    in headerValid `mappendLefts` subValid
+        ptxValid    =
+            foldl' (\acc ptx -> validatePtx ptx
+              `mappendLefts` acc)
+                pass subs
```

Comment:
:cry:

Reply:
Is that true that you use some tool for automatic formatting to get those aligned `=` and indentation like for `ptxValid`? (I guess `hindent`?)

Reply:
I use my own hands. However, I didn't recall myself editing this that way, I'll look.

Reply:
Okay, I see. Lol